scripts/anonymization/anonymizer.py

```diff
@@ -0,0 +1,210 @@
+"""Module that handles the full Anonymization pipeline
+"""
+
+import sys
+import xml.etree.ElementTree as ET
+
+
+def rewriteFakeQueries(fakeValues, path):
+    """Method that rewrites existing query templates based on sensitive value replacements
+    """
+    # TODO
+
+
+def fakeColumn(dataset, col, locales, method, seed=0):
+    """Method that generates fake values for columns that are considered sensitive
+    """
+    # TODO
+
+
+def getTimestampColumns(dbTypeList):
+    """A helper function that returns a list of indexes of timestamp-type columns
+    """
+    # TODO
+
+
+def dfFromTable(curs, table):
+    """Helper function that creates a pandas DataFrame from a jaydebe connection
+    """
+    # TODO
+
+
+def populateAnonFromDF(curs, df, table, timestampIndexes):
+    """Helper function to fill a DB table from a DataFrame
+    """
+    # TODO
+
+
+def getDroppableInfo(dropCols, dataset):
+    """Helper function that saves droppable columns from anonymization
+    """
+    # TODO
+
+
+def getTransformer(
+    dataset, algorithm, categorical, continuous, ordinal, continuousConfig
+):
+    """Function that creates a custom transformer for the dp-mechanism
+    """
+
+    # TODO
+
+
+def getConstraints(objects):
+    """A helper method that builds constraints that are used by a TableTransformer
+    """
+    # TODO
+
+
+def anonymize(
+    dataset: str,
+    anon_config: dict,
+    sens_config: list,
+    cont_config: list,
+    templatesPath: str,
```

Comment:
Is it possible to set typing hints for all of the functions?  Or leave that until you fill in the details later on?